Or 857/align typography documentation (#1816)
Financial-Times/origami: apps/for-everyone-website/src/content/docs/guides/typography.mdx

<BrandedContent brands="core, professional, sustainable-views, whitelabel">
	### Financier Display
	Financier is a secondary typeface. Elegant, authoritative and has a distinct voice.
	Primarily used for large article titles and headings in editorial pages only.

    [Learn more about Financier Display](https://brand.ft.com/document/24#/branding-elements/fonts).

</BrandedContent>

<BrandedContent brands="core, professional, whitelabel">

    ### Georgia
    Georgia is solely used for editorial article body copy. This is an alternate to Financier Text, as a [system
    font](https://en.wikipedia.org/wiki/Web_typography) it avoids the performance implications of users downloading
    additional custom fonts.

</BrandedContent>

<BrandedContent brands="sustainable-views">
	### Financier Text
	Financier Text is our main editorial typeface and is used predominantly in our journalism. It is elegant and
	authoritative, has a distinct voice, and works well across long copy formats.

    [Learn more about Financier Text](https://brand.ft.com/document/24#/branding-elements/fonts).

</BrandedContent>

## Typographic Scale

> We use a single typographic scale for all fonts. Our scale ranges from -2 to 12. Consider scale 0 the baseline, with ascending and descending values in each direction. Avoid using styles below zero for blocks of text premium user experience.
> 
> - Our legacy typographic scale uses the same values as our latest scale. See our legacy Foundations library and legacy o-typography component in Storybook.

Reviewer: This isn't accurate:
1. We have multiple scales
2. For now Metric 2 is -3 to 9, others -2 to 10

```suggestion
We use multiple typographic scales for different font families. Our scale ranges from -3 to 10, depending on the font family. Consider scale 0 the baseline, with ascending and descending values in each direction. Avoid using styles below zero for long form content.
```

> ### Metric 2 (Variable Font, VF)
> 
> Modern Origami introduces the Metric2 font family as a variable font (a incremental font weights). We use a single typographic scale for all fonts.
> 
> Metric2 replaces legacy Metric font files. Metric2 renders larger than Metric for the same font size, therefore you may need to use a smaller scale to align with legacy Metric typography.
> 
> ![A comparison of fonts Metric and Metric 2. Font size on Metric 2 is 8 pixels less than its former. And its line height stays the same](/assets/images/components/typography/metric-metric2-diff.svg)

Reviewer: This is great but will be out of date in a few months (when metric2 is the default). May I propose deleting for now and moving it to onboarding pages "what's new in o3" and our onboarding workshops, in the coming weeks?

```suggestion
```

## Font Weight

> Font weight can be either be used to express hierarchy or urgency of information.

Reviewer: There are a few issues with this section:
1. It does not apply to every brand. The examples are very article orientated, maybe even ft.com specific, and should vary per brand accordingly.
2. I don't think the examples add to our understanding of how to use weight for emphasis.
3. We should be discouraging direct use of our scale and weights, instead pointing to uscases, which this doesn't do where it did before.

I'd propose reverting this as it's out of scope of the mobile type optimisation and returning to this when Mary and Deyan are back

| <div class="not-content"><p class="o3-typography-body-standard">Body Standard - Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p></div>       | Body standard is the default text size. Used for detailed or descriptive content, in components, or where space is limited.                    |
| <div class="not-content"><p class="o3-typography-body-small">Body Small - Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p></div>             | Body standard is the default text size. Used for detailed or descriptive content, in components, or where space is limited.                    |
| <div class="not-content"><p class="o3-typography-body-small-bold">Body Small (bold) - Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p></div> | Body standard is the default text size. Used for detailed or descriptive content, in components, or where space is limited.                    |

> | <div class="not-content"><p class="o3-typography-body-small-caps">Body small (caps) - Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p></div> | Body standard is the default text size. Used for detailed or descriptive content, in components, or where space is limited.                    |

Reviewer: "Body standard" is repeated here... and is lorem ipsum

> <BodyTypography brand={props.brand} />

Reviewer: I like the way you've approached a table of content and grouped preview for the code snippet. I'm not sure it shows the relationship between styles well (body small is the same, but uppercase and bold is avaliable). Let's revisit that when Mary is back

### Links

> For general links in UI pages. Link style can be applied inline (within a paragraph or body text) or standalone. The inline links inherits the text properties of the body it is in.

Reviewer: ```suggestion
For general links in UI pages. The link style can be applied inline (within a paragraph or body text) or standalone. The inline links inherits the text properties of the body it is in.
```

Link focus style follows the global Origami focus for single ring element.

<LinkTypography brand={props.brand} />

**Use descriptive words to describe links**

> Link text should be clear, concise, and descriptive enough to make sense out of context. Avoid vague phrases like "click here" or "read more." for example: use "Read our accessibility guidelines" Instead.

Reviewer: This is the only example where we put copy after the preview –  which makes it easy to overlook in my view – and used bold text to highlight an important usage guideline. Can we move this to the useage guidelines at the bottom of the as a do/don't? 

We may then want to chat with Mary later about how moving these do/don'ts closer to the right sections

### Footer

> 
> <FooterTypography brand={props.brand} />

Reviewer: Bringing back a temporary short description from the current site

```suggestion
For footers and copyright notices.

<FooterTypography brand={props.brand} />
```

### Caption

A caption is a brief text description or explanation that provides context to visual elements such as images, illustrations, charts, or infographics. Captions should be kept brief.

<CaptionTypography brand={props.brand} />

### List

For lists in UI pages. Lists allow users to view two or more individual, but related, text items grouped together. We have ordered and unordered lists.

<ListTypography brand={props.brand} />

<BrandedContent brands="core, professional, sustainable-views, whitelabel">
	## Editorial Typography

>     Editorial typography is intended for editorial and article content e.g homepage and Content pages. For typography to use for UI, visit here.The following font families are used when designing for editorial.
> 
> Keeping typography consistent and sticking to logical hierarchies ensures that page elements are clear and easily recognisable when scanning the page. Lead with Mobile-First (also applies to FT Apps. For example; FT Apps, FT Edit)

Reviewer: - "Visit here" is supposed to be a link, but also we need to follow our own guidelines and make it descriptive 😅 
- "The following font families are used when designing for editorial." doesn't follow with font families

Proposal to stick with what was there. I think this may be an unintentional regression after previous iterations on content.

```suggestion
Editorial typography is intended for editorial and article content e.g article and home pages, live news, teasers.
```

    	<Code lang="css"
    				code={`:root{\n\t--o3-editorial-typography-topic-tag-color: hotpink;\n\t--o3-editorial-typography-topic-tag-hover-color: deeppink;\n}`} />
    </details>

    <TopicTag brand={props.brand} />

    ### Standfirst
    Short intro below the headline that provides context about a post. Used in
    combination with a header in the topper component.
    <StandFirst brand={props.brand} />

    ### Caption
    For additional information about a piece or image that should be less
    prominent.
    <Caption brand={props.brand} />

    ## Usage Guidelines

Keeping typography consistent and sticking to logical hierarchies ensures that elements in the UI are clear and easily recognisable when scanning the page.

<Guideline summary="Define heading levels" do={true}>
	<p slot="description">
		Styles can be assigned a heading level that is independent of style name,
		size, weight, or other properties.
	</p>
	<Example slot="examples" do={true}>
		![A h1 heading follows a h2
		heading.](/assets/images/components/typography/type-guideline-1.svg)
	</Example>
</Guideline>

    <Guideline summary="Don’t use Financier fonts for UI" do={false}>
    	<p slot="description">

>     		Financier display font should be used for Editorial article content only, use metric font instead for UI.

Reviewer: ```suggestion
    		The Financier Display font must be used for Editorial article content only, use Metric for UI.
```

    <Guideline summary="Avoid negative font scales" do={false}>
    	<p slot="description">

>     		Scales less than zero,  `o3-font-size--1`(14px) and `o3-font-size--2`(12px), are edge cases. Avoid use as they can be hard to read especially on large screens. Use to show less important information such as [Captions](#caption) etc

Reviewer: There are good / bad points for the before and after. Let's combine and hint that we're moving away from these entirely in the future.

```suggestion
    		Scales less than zero,  `o3-font-size--1`(14px) and `o3-font-size--2`(12px), are edge cases. Avoid these as they can be hard to read especially on large screens. Negative scales may be used to present short, secondary information where needed to align to existing designs. However we recommend using predefined [ui typography](#ui-typography) styles where possible.
```

    </Guideline>

</BrandedContent>

> 
> ## Accessibility notes
> 
> - ARIA Roles: Ensure lists are properly identified by screen readers using appropriate HTML tags (`<ol>` for ordered lists and `<ul>` for unordered lists).
> - Keyboard Navigation: Lists should be navigable via keyboard.

Reviewer: I think some of this content might be stale 🤔 Me and Mary previously agreed not to add this as:
- It's not poss

```suggestion
- Using semantic markup is a given using our components and code snippets
- Keyboard navigation is automatic (unless you try hard to break it)
```
